packages/e2e-tests/prepare.ts

```diff
@@ -21,4 +21,5 @@ async function run(): Promise<void> {
   }
 }
 
-void run();
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+run();
```

Comment:
Wdyt?
```suggestion
(async () => { await run() })();
```

Reply:
This will still create a floating promise